x-pack/legacy/plugins/actions/server/builtin_action_types/es_index.ts

```diff
@@ -52,9 +53,15 @@ async function executor(execOptions: ActionTypeExecutorOptions): Promise<ActionT
   const services = execOptions.services;
 
   if (config.index == null && params.index == null) {
+    const message = i18n.translate('xpack.actions.builtin.es_index.indexParamRequired', {
```

Comment:
Mixing camel and snake case here.  These should all be camel case.  Same with all the message keys in this PR.

Reply:
resolved in subsequent commit